libs/entry-components/styles/modules/vendors/redactor/_tables-generator.scss

```diff
@@ -0,0 +1,29 @@
+@use 'sass:map';
+@use '../../../../../scss-foundation/src/modules/lists/row-coloring' as list;
```

Comment:
@typhoon41 @Cvetanovic  I used a mixin from scss-foundation here, but I’m concerned about what will happen if the scss-foundation package isn’t installed at the project level. Should I create a custom mixin specifically for Redactor row coloring to be safe? I also haven't found a way to avoid using relative paths